frame/contracts/src/exec.rs

```diff
@@ -1053,7 +1058,16 @@ where
 		}
 
 		let value = frame.value_transferred;
-		Self::transfer(ExistenceRequirement::KeepAlive, self.caller(), &frame.account_id, value)
+
+		// Get the account id from the Caller.
```

Comment:
```suggestion
		// Get the account id from the caller.
```
let's keep these the same here and in all other added comments: either `Caller` or `caller`, either `AccountId` or `account id`